pytest_ansible/fixtures.py

```diff
@@ -1,61 +1,39 @@
-import pytest
-import warnings
-import logging
-from .errors import AnsibleHostUnreachable
-
-try:
-    from logging import NullHandler
-except ImportError:
-    from logging import Handler
+"""PyTest fixtures."""
 
-    class NullHandler(Handler):
+import pytest
+from pytest_ansible.logger import get_logger
 
-        def emit(self, record):
-            pass
+__all__ = ('ansible_module', 'ansible_facts', 'ansible_adhoc', 'localhost')
+log = get_logger(__name__)
 
-log = logging.getLogger(__name__)
-log.addHandler(NullHandler())
 
+@pytest.fixture(scope='function')
```

Comment:
Yup, I'd like to merge massive set of commits now, and we'll follow-up in a future PR with a more concise change for that (and a few other things raised during discussion in this PR).